Offseason2022/src/main/java/frc/robot/commands/Climber7ClimbToL4.java

```diff
@@ -13,17 +16,18 @@ public class Climber7ClimbToL4 extends SequentialCommandGroup
 {
   public Climber7ClimbToL4(Climber climber)
   {
+    setName("ClimberClimbToL4");
+
     addCommands(
-    // Add Commands here:
-    // Also add parallel commands using the
-    //
-    // addCommands(
-    // new command1(argsN, subsystem),
-    // parallel(
-    // new command2(argsN, subsystem),
-    // new command3(argsN, subsystem)
-    // )
-    // );
+        // Add Commands here:
+
+        // @formatter:off
+      new ParallelDeadlineGroup(
+          new WaitUntilCommand(climber::moveClimberDistanceIsFinished),
+          new ClimberMoveToHeight(climber, Height.RAISE_L4_HEIGHT), 
+      new ClimberSetGatehook(climber, false)

```

Comment:
Same comment as previous. All of the commands enclosed within the ParallelDeadlineGroup should have the same indent level.